cfgov/jinja2/v1/_includes/blocks/feedback.html

```diff
@@ -6,15 +6,18 @@
 
 {# JS-disabled result handling #}
 
-
-<div class="o-feedback
-            block
-            {% if value.was_it_helpful_text %}
-                block__bg
-                block__border
-                block__padded-top
-            {% endif %}">
-    <div class="u-mb15">
+<div class="o-feedback block">
+	<form method="post"
+	      action="."
+	      class="o-form
+	      		 {% if value.was_it_helpful_text %}
+	      		 block
+		         block__bg
+		       	 block__border
+		       	 block__padded-top
+	       		 {% endif %}"
+	      novalidate>
+	<div class="u-mb15">
```

Comment:
Could this use some combination of `block`? I think `block block__sub block__flush-top` may do it. That's off the top of my head though.

Reply:
I have no real opinion on this, but does something in this change require those new classes? Was that just an outstanding issue with this code that it should have had eg, `block__sub` and `block__flush-top` and it would be convenient to sneak it in here?

Reply:
Yeah, may be out of scope for here so feel free to ignore it. I thought we can set 15px margin with blocks, though it may only be 15px padding that can be set and 30px margin. `block` combinations are very verbose, but we probably want to be consistent with using them and then could swap in abbreviated patterns for all of them if we ever get to that. Food for thought.